azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/VirtualNetworkGateway.java

```diff
@@ -63,6 +63,20 @@
     @Method
     Observable<VirtualNetworkGatewayConnection> listConnectionsAsync();
 
+    /**
+     *  Generates asynchronously VPN profile for P2S client of the virtual network gateway in the specified resource group. Used for IKEV2 and radius based authentication.
```

Comment:
The comments here and the one for the method below should be reversed.

Reply:
Also it might be goo to break it into two sections, one short description followed by a paragraph and the rest of the description.